src/components/Navbar/SettingsDrawer/TranslationSection.module.scss

```diff
@@ -6,3 +6,8 @@
   flex-direction: column;
   margin-block-start: var(--spacing-xsmall);
 }
+
+.skeleton {
+  height: calc(1.5 * var(--spacing-mega));
```

Comment:
Salam brother @abdulhamidOumer . JazakAllah for the contribution. 

I just did a quick test. On my safari macbook, it seems that we need a bigger height?

For a reference, this 1.5 * spacing mega skeleton side by side with the loaded component
<img width="426" alt="Screen Shot 2022-10-04 at 13 37 12" src="https://user-images.githubusercontent.com/12745166/193750840-a376cf42-10ae-4266-b4a1-bef9938763d4.png">

and this is 2.2
<img width="414" alt="image" src="https://user-images.githubusercontent.com/12745166/193750949-90755616-c218-43b3-9ba7-370d7994e320.png">

other than that, looks good Masha Allah

Reply:
Thank you for the review @muhajirdev 👍 . You're right when it's multiplied by `1.5` the skeleton height is smaller than what will be rendered. As suggested multiplying it with `2.2` matches the height somehow. 

I have pushed a fix for that.